docs/docs/typechecking-with-proptypes.md

```diff
@@ -56,6 +56,9 @@ MyComponent.propTypes = {
 
   // A React element.
   optionalElement: PropTypes.element,
+  
+  // A function or class
+  optionalElement: PropTypes.func
```

Comment:
It's not valid to define the same property twice. Instead use `PropTypes.oneOfType([...])`